tools/scripts/import_whitequark.sh

```diff
@@ -14,8 +14,8 @@
 set -euo pipefail
 set -x
 
-REF=2a705681cfe339261b69b03593d008cb2c4eff35
-TARGET_RUBY_VERSION="2.6"
+REF=b328d5f4d38e5347feb91aa9e55157c858b07cc6
```

Comment:
Might change again later if more tests are added regarding 2.7.

Reply:
Also, I didn't commit all the tests imported right away. I'll add them in subsequent pull-requests for each new feature.